onnx/defs/math/defs.cc

```diff
@@ -297,22 +347,30 @@ produces one output data (Tensor<T>) where the function `f(x) = x^exponent`,
 is applied to the data tensor elementwise.
 )DOC" + std::string(kBroadcastDoc))
     .Input(0, "X", "Input tensor of any shape, base of the exponent.", "T")
-    .Input(1, "Y", "Input tensor of any shape broadcastable to X shape, "
-                   "the exponent component.", "T")
-    .Attr("broadcast",
-          "Pass 1 to enable broadcasting",
-          AttributeProto::INT,
-          static_cast<int64_t>(0))
-    .Attr("axis",
-          "If set, defines the broadcast dimensions. See doc for details.",
-          AttributeProto::INT,
-          OPTIONAL)
+    .Input(
+        1,
+        "Y",
+        "Input tensor of any shape broadcastable to X shape, "
+        "the exponent component.",
+        "T")
+    .Attr(
+        "broadcast",
+        "Pass 1 to enable broadcasting",
+        AttributeProto::INT,
+        static_cast<int64_t>(0))
+    .Attr(
+        "axis",
+        "If set, defines the broadcast dimensions. See doc for details.",
+        AttributeProto::INT,
+        OPTIONAL)
     .Output(0, "Z", "Output tensor (same size as X)", "T")
-    .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
+    .TypeConstraint(
+        "T",
+        {"tensor(float16)", "tensor(float)", "tensor(double)"},
         "Constrain input and output types to float tensors.");
 
 ONNX_OPERATOR_SCHEMA(PRelu)
-    .AllowConsumed({{0, 0}})
+    .SinceVersion(7)
```

Comment:
As we discussed, this should be .SinceVersion(6), (because in my previous PR, we already bump up the opset version, no need to do it again, otherwise, I have to prepare opset_7 model zoos, which does not exist yet... why our test case does not fail on this?